experiment/simulation/css/main.css

```diff
@@ -1 +1,250 @@
+}
+
+table {
+    border: 2px solid #ccc;
+    border-collapse: collapse;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    border-width: medium;
+    line-height: 210%;
+    border-color: #673939;
+    min-width: 150px;
+}
+
+table tr {
+    background-color: #f8f8f8;
+    border: 2.5px solid #673939;
+}
+
+table td {
+    font-size: 85%;
+    text-align: center;
+    position: relative;
+    border: 1.25px solid #673939;
+}
+
+table th {
+    font-size: 85%;
+    text-align: center;
+    border: 2px solid #673939;
+}
+
+table.table.table-bordered {
+    background-color: #ffffff;
+    border-width: medium;
+    border-color: #673939;
+}
+
+div.operations button {
+    font-family: monospace;
+    font-weight: bold;
+    width: 9rem;
+    text-align: left;
+}
+
+.table-bordered thead th {
+    border-bottom-color: #673939;
+    border-bottom-width: 100%;
+}
+
+.table-bordered td {
+    border: 2px solid #673939;
+}
+
+.rtable.table-bordered {
+    border-width: medium;
+    border-color: #673939;
+}
+
+#five-switch a {
+    color: white;
+    text-decoration: none;
+}
+
+#four-switch a {
+    color: white;
+    text-decoration: none;
+}
+
+#main-heading {
+    align-content: center;
+    text-align: center;
+    font-weight: 600;
+    font-size: 2.3vw;
+    color: #363636;
+}
+
+#result-heading {
+    font-weight: bold;
+    font-size: 2vw;
+    color: #363636;
+    /* min-width: 280px; */
+}
+
+.message {
+    text-align: center;
+    background-color: rgb(164, 198, 82);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1.5vw;
+    color: white;
+}
+
+.br {
+    display: block;
+    margin-bottom: 2em;
+}
+
+.btn {
+    border-radius: 16px;
+    border: none;
+    font-family: sans-serif;
+    text-align: left;
+    font-size: 0.7vw;
+    background-color: #288EC8;
+    width: 100px;
+    color: white;
+}
+
+.btn:hover {
+    background-color: #A9A9A9;
+}
+
+#anku {
+    font-size: 90%;
+}
+
+@media screen and(max-width: 769px) {
+    #main-heading {
+        padding-top: 10px;
+        text-align: center;
+        font-size: 5.4vw;
+    }
+    .message {
+        font-size: 4vw;
+        margin-left: 10px;
+        margin-right: 10px;
+    }
+    table th {
+        font-size: 75%;
+        text-align: center;
+        font-weight: 600;
+    }
+    table td {
+        font-size: 75%;
+        text-align: center;
+    }
+    #anku {
+        font-size: 85%;
+    }
+    #result-heading {
+        font-size: 5vw;
+        font-weight: bold;
+    }
+    .btn {
+        font-size: 60%;
+        width: 44%;
+    }
+}
+
+#result {
+    min-width: 100px;
+}
+
+@media screen and (max-width: 1570px) {
+    .message {
+        font-size: 2vh;
+        margin-left: 10px;
+        margin-right: 10px;
+    }
+    #main-heading {
+        padding-top: 10px;
+        font-size: 3vh;
+    }
+    #result-heading {
+        font-size: 3vh;
+    }
+    .btn {
+        font-size: 70%;
+    }
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
Why do we need 2 blocks one for max-width 768px and another for max-width 769px. What changes in 1 pixel that justifies a whole new block of code?